### PR TITLE
Lower password size to 10

### PR DESCRIPTION
**Pull request by SebastienGllmt**
This matches both Daedalus and the Yoroi extension which both only require 10

### CODE REVIEW

File: src/utils/validators.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const getPasswordStrength = (password: string): PasswordStrength => {
     return {isStrong: false}
   }
 
-  if (password.length >= 12) {
+  if (password.length >= 10) {
     return {isStrong: true, hasTwelveCharacters: true}
```
Comment (v-almonacid) [inline src/utils/validators.js]: Maybe rename `hasTwelveCharacters` by something more generic like `hasMinimumLen` ?

File: src/components/WalletInit/PasswordStrengthIndicator.js [hunk 2]
```diff
@@ -21,34 +20,14 @@ const messages = defineMessages({
...
     defaultMessage: '!!!Continue',
     description: 'some desc',
   },
   passwordBigLength: {
     id: 'components.walletinit.passwordstrengthindicator.passwordBigLength',
-    defaultMessage: '!!!12 characters',
+    defaultMessage: '!!!10 characters',
     description: 'some desc',
   },
   or: {
```
Comment (v-almonacid) [inline src/components/WalletInit/PasswordStrengthIndicator.js]: This one also seems unnecessary now (sorry 😬)

Comment (SebastienGllmt) [reply src/components/WalletInit/PasswordStrengthIndicator.js]: How is it unnecessary? It's used to show the user the current password requirement is 10 characters.

Comment (v-almonacid) [reply src/components/WalletInit/PasswordStrengthIndicator.js]: I mean the "or" string, I believe it's not used anymore.

Comment (SebastienGllmt) [reply src/components/WalletInit/PasswordStrengthIndicator.js]: A good point. Removed some more dead code

**Review by v-almonacid — APPROVED** [review]
LGTM

**Pull request merged**